osprepare/distro.go

```diff
@@ -229,6 +229,7 @@ func sudoFormatCommand(command string, packages []string) bool {
 	c := exec.Command(executable, strings.Split(args, " ")...)
 	c.Stdout = os.Stdout
 	c.Stderr = os.Stderr
+	c.Env = os.Environ()
```

Comment:
Good question.  The original bug report didn't have enough details for me to understand how their system was configured.

The "If Env is nil, Run uses the current process's environment." comment and if nil return os.Environ() has been in os/exec/exec.go since 2011.

@albertomurillo can you confirm how you're running and failing in issue #511 and whether this patch resolves for you?  Based on @rbradford's archaeology, we're thinking this patch doesn't actually change anything.